client-web/src/exlibris-app/menu/exlibris-menu-item.html

```diff
@@ -58,6 +57,7 @@
          * `--menu-item-title-color` | Element text color | black
          * `--menu-item-background-color` | Background color of the element display area | white
          * `--exlibris-menu-item-border-radius` | Item border radius | 3px
+         * `--exlibris-menu-item-title-font-weight` | Item title font weight | normal
```

Comment:
I see 2 css properties for the same thing: title. Maybe it's better to use css mixin?